test/conformance/testing/include/uur/utils.h

```diff
@@ -170,7 +170,18 @@ inline std::string GetDeviceName(ur_device_handle_t device) {
 }
 
 inline std::string GetPlatformAndDeviceName(ur_device_handle_t device) {
-    return GetPlatformName(GetPlatform()) + "__" + GetDeviceName(device);
+    char pciAddress[256];
+    char id[256];
+    size_t propSize = sizeof(pciAddress);
+    urDeviceGetInfo(device, UR_DEVICE_INFO_PCI_ADDRESS, propSize, pciAddress, &propSize);
```

Comment:
As you don't verify the actual returned size of the property, I suggest leaving the last argument as the `nullptr`.